cmake/Dependencies.common.cmake

```diff
@@ -263,6 +263,8 @@ endif()
 if (BUILD_CVCUDA)
   set(DALI_BUILD_PYTHON ${BUILD_PYTHON})
   set(BUILD_PYTHON OFF)
+  # for now we use only median blur from CV-CUDA
+  set(CV_CUDA_SRC_PATERN medianblur median_blur)
```

Comment:
It's "PATTERN"

Reply:
I'm afraid that I made that typo in CVCUDA as well - https://github.com/CVCUDA/CV-CUDA/blob/release_v0.4.x/src/cvcuda/CMakeLists.txt#L75. So it works although there is a typo.